git-clone-repos.yaml

```diff
@@ -1,42 +1,45 @@
 ---
-- name: Playbook that setup's the repositories of an organisation for the User in local.
+- name: Playbook that setup's the repositories of an organisation for the USER in local.
```

Comment:
This does not parse... 'sets up' ... 'for the user given as `USER`' ?

Reply:
yes, it doesn't parse the USER!
As the user can have a different username on the local system and GitHub. So changed the variable to GITHUB_USER and prompt a msg if the user doesn't specify GITHUB_USER. Also skips the git-remote setup.